README.InfluxDB.md

```diff
@@ -13,3 +13,12 @@ Set `"api"` to`"influxdb"` to use the InfluxDB API.
 
 For optional parameters such as `username` or `password` have a look at the
 example config file is available at [`etc/vzlogger.conf.InfluxDB`](https://github.com/volkszaehler/vzlogger/blob/master/etc/vzlogger.conf.InfluxDB)
+
+Message Format
+---------------------------
+
+Messages are sent in the following format:
+
+<measurement_name>,uuid=<uuid>,<tags> value=<value> <time[ms]>
```

Comment:
is this more useful than,
or shouldn't it at least contain a pointer to,
the official documentation:
https://docs.influxdata.com/influxdb/v1.8/write_protocols/line_protocol_tutorial/